src/main/kotlin/com/perquizz/perquizz/migrations/services/MigrationServiceImpl.kt

```diff
@@ -0,0 +1,21 @@
+package com.perquizz.perquizz.migrations.services
+
+import com.perquizz.perquizz.migrations.dtos.MigrationsResponseDto
+import com.perquizz.perquizz.migrations.repository.MigrationRepository
+import org.springframework.stereotype.Service
+
+@Service
+class MigrationServiceImpl(val repository: MigrationRepository) : MigrationService {
+    override fun getPendingMigrations(): MigrationsResponseDto {
+        return MigrationsResponseDto(repository.findPendingMigrations())
+    }
+
+    override fun migrate(): MigrationsResponseDto {
```

Comment:
```suggestion
    override fun migrate(): MigrationsResponseDto =
        repository.findPendingMigrations()
            .also { repository.migrate() }
            .let { MigrationsResponseDto(it) }
```
Assim conseguimos remover a chamada do repository de dentro do parametro do dto, e a remoção de chaves utilizando função de escopo do kotlin o que acha ?